data/json/items/vehicle/utilities.json

```diff
@@ -214,6 +214,26 @@
       }
     ]
   },
+  {
+    "type": "GENERIC",
+    "copy-from": "washing_machine",
+    "id": "fridgetank_large",
+    "name": { "str": "refrigerated tank (100L)", "str_pl": "refrigerated tanks (100L)" },
+    "description": "A 100L refrigerated tank for keeping liquids cool.  Provides some insulation from outside weather.",
+    "looks_like": "fridgetank",
+    "insulation": 10,
+    "volume": "112 L",
+    "weight": "67 kg",
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "watertight": true,
+        "max_contains_volume": "100 L",
+        "max_contains_weight": "500 kg"
+      }
+    ]
+  },
```

Comment:
This will be material steel ingame but is made of aluminum in the recipe. Don't.